### ISSUE TITLE
Request - Adding page for classic collection

### ISSUE THREAD

**Issue by vrnvorona** [Js41637/Overwatch-Item-Tracker#148]
There are a lot of event-related pages, but what about classic game page? It would be easier for people to check items without switching heroes knowing that they have all sprays for non-event collection etc. 
Because new loot system gives you a duplicate only if you have all items it that category.

**Comment by Js41637** [comment]
You mean a page that has every item in the game on it? There is too many items in the game to add them all to one page?

**Comment by vrnvorona** [comment]
No, every but event. Like default skins. Or like some categories like "Voice lines" etc. Idk. If so bad ik, nvm.

**Comment by francoisgervais** [comment]
I could see myself using this, and since you raised the point, I wouldn't mind a section with every item in game that I could filter down (ie an easy way to see all unowned highlight intro, see all owned legendary skins, etc) to plan purchases or view global progress.

Awesome work on this project by the way, I love it!
